base/ltmarks.dtx

```diff
@@ -672,8 +786,8 @@
 %    does not update mark structures and can therefore be used outside
 %    the output routine as well.
 %
-%    It collect all the top-level marks from inside the \meta{source}
-%    and adds suitable \cs{mark_insert:nn} in the two token
+%    It collects all the top-level marks from inside the \meta{source}
+%    and then it adds suitable \cs{mark_insert:nn} in the two token
```

Comment:
?? Does this mean:
. . . adds a suitable . . . to each of the two . . . 
Or:
. . . adds suitable . . . commands to each of the two . . .